templates/footer.html

```diff
@@ -0,0 +1,38 @@
+<footer class="page-footer">
+    <div class="container">
+        <div class="row">
+            <div class="col m3 s12">
+                <h5 class="white-text fine">{{ org_name }}</h5>
+                <ul>
+                    <li><a class="grey-text text-lighten-3" href="http://{{ org_name }}.io/docs">Documentation</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://{{ org_name }}.io/newcomer">Newcomer Docs</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://{{ org_name }}.io/coc">Code of Conduct</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://blog.{{ org_name }}.io/">Blog</a></li>
+                    <li><a class="grey-text text-lighten-3" href="mailto:sponsoring@{{ org_name }}.io">Donate</a></li>
+                </ul>
+            </div>
+            <div class="col m9 s12">
+                <div class="row footer-btn-row">
+                    <div class="col m6 s12 center">
+                        <div class="twitter-btn center">
+                            <a href="https://twitter.com/{{ org_name }}_io" class="waves-effect waves-light btn-large"><i class="fa fa-twitter" aria-hidden="true"></i><span class="footer-btn-text">Follow Us On Twitter</span></a>
```

Comment:
IMO doing stuff like this is completely against the point of not using the word `coala` as this will only work for us, not any other org. This might have already been discussed and will be fixed later, just wanted to provide my input on this.

Reply:
Exactly. We’ll have to create a config file. But as discussed, we’ll come up with config file in next PR